src/eko/basis_rotation.py

```diff
@@ -355,7 +355,7 @@ def select_light_flavors_uni_ev(ad_lab, nf):
         return map_ad_to_evolution[ad_lab]
 
 
-def ad_projectors(nf, qed=False):
+def ad_projectors(nf, qed):
```

Comment:
Are these functions still used somewhere?

Reply:
I don't think so

Reply:
We always said that EKO is a library in the first place, and not an application. So, not all the functions have to be used.

For as long as they are correct, and they do not introduce much extra burden, I would keep them. It was non-trivial to get the normalization right (maybe because we're not especially clever, but to us, it was confusing).

They are just part of the *basis rotation toolbox*.

Reply:
okay, it make sense, let's keep them.